### ISSUE TITLE
Mergear y solucionar problemas de los 2 forks del proyecto 2020-WCCI-variable-attack-surface

### ISSUE THREAD

**Issue by rafaelleru** [rafaelleru/tfg#1]
El codigo del repositorio fue forkeado y hay 2 versiones del mismo. 

El objetivo de esta historia de usuario es unificar los codigos de los 2 repositorios:

* https://github.com/JJ/2020-WCCI-variable-attack-surface
* https://github.com/JJ/2020-evo-mtd-ppsn 

Y hacer que el proyecto pueda correr sin problemas.

**Comment by rafaelleru** [comment]
@JJ es probable que no vieras este issue, como comentamos es la fusion de los 2 forks.

**Comment by JJ** [comment]
Cierto, no lo vi. En todo caso, recuerda que los issues deben ir siempre ligados a una historia de usuario y encuadrados en un milestone. Diseña un milestone para esto, con su PMV, y su HU correspondiente.

**Comment by rafaelleru** [comment]
Este issue completara la historia de usuario #3 una vez completado.
